samples/node-token-server/README.md

```diff
@@ -6,18 +6,19 @@ Express server that generates Coveo search tokens for Dynamics portal.
 
 Before running the sample, open `Index.ts` and change this configuration accordingly to represent your environment.
 
-``` javascript
+```javascript
 const config = {
-    portalUrl: "<your_portal_url>", // example: https://yourportalurl.microsoftcrmportals.com/
-    coveoApiKey: "<your_API_key>", // The API key used to query Coveo and create a search token. It must have at least the privileges "Execute query" and "Impersonate" enabled.
+    portalUrl: "<your_portal_url>", // Example: https://yourportalurl.microsoftcrmportals.com (without slash at the end)
```

Comment:
I suggest we handle the ending slash in the code. It is pretty annoying when a piece of software breaks because of an assumption like that.